app/i18n/en/namespaces/feed.js

```diff
@@ -2,9 +2,12 @@
 
 const feed = {
   feed_item: {
-    comment: ' {{action}} {{object}}',
-    deleteTopic: ' {{action}} {{object}}',
-    createTopic: ' {{action}} {{object}}',
+    action_COMMENT: ' commented on topic ',
+    action_CREATE: ' created the topic ',
+    action_DELETE: ' deleted the topic ',
+    action_FORK: ' forked the topic ',
+    action_UPDATE: ' updated the topic ',
```

Comment:
A bit of a nitpick, but I think its best to keep starting & trailing spaces out of translated strings; add the space in the place where you include the string instead. Otherwise it's way too easy for translators to accidentally forget the spaces.